CONTRIBUTING.rst

```diff
@@ -120,6 +120,29 @@ Running Tests
       cd ~/hack-on-artman
       nox
 
+-   To run the full set of artman smoke tests, you need to install docker
+    installed on your machine, and run the following at the root of your
+    artman source directory:
+
+  .. code::bash
+
+      docker pull googleapis/artman:latest
+      docker run -it \
+      -v ${PWD}:/usr/src/artman \
```

Comment:
Not a blocker, but a two-space hanging indent on line-continued commands would be nice.